### PR TITLE
python3 compatibility

### PR DESCRIPTION
**Pull request by zebuline**
* python3 compatibility
* restore testing of `demo/embeddedsigning.py` and `demo/templates.py`
* use requests instead of pycurl for envelope creation, for python3 compatibility (and to simplify the code)
* `demo/accounts.py` anonymized
* (novapost renamed into novafloss)

### CODE REVIEW

**Comment by zebuline** [comment]
ping @tbcvl @k4nar @bersace

File: pydocusign/models.py [hunk 1]
```diff
@@ -729,10 +730,8 @@ def get_recipients(self, client=None):
             recipient.roleName = recipient_data.get('roleName', None)
             synced_recipients.append(recipient)
 
-        def cmp_recipients(a, b):
-            return cmp(a.routingOrder, b.routingOrder)
-
-        synced_recipients = sorted(synced_recipients, cmp_recipients)
+        synced_recipients = sorted(
+            synced_recipients, key=attrgetter('routingOrder'))
```
Comment (k4nar) [inline pydocusign/models.py]: :+1:

Comment (k4nar) [inline pydocusign/models.py]: we could use .sort to do it in-place though

File: pydocusign/client.py [hunk 2]
```diff
@@ -252,112 +250,46 @@ def delete_account(self, accountId):
...
-            'url': url,
-            'headers': headers,
-            'body': body,
-        }
+        data['documents'] = [{
+            "documentId": "1",
+            "name": "document.pdf",
+            "fileExtension": ".pdf",
+            "documentBase64": base64.b64encode(
+                document.read()).decode('utf-8')}]
```
Comment (k4nar) [inline pydocusign/client.py]: Does Docusign recommend the base64 version over the multipart/form-data one?

File: demo/accounts.py [hunk 3]
```diff
@@ -80,13 +80,13 @@ def prompt(environ_key, description, default):
...
 print("3. POST /accounts")
-account_email = "benoit.bryon+pydocusign-test@novapost.fr"
+account_email = "benoit.bryon+pydocusign-test@people-doc.com"
 account_password = "notasecret"
 account_input = {
     "accountName": "Pydocusign Test Account",
     "accountSettings": [],
     "addressInformation": {
-        "address1": "32 rue de Paradis",
+        "address1": "53 rue d'Hauteville",
```
Comment (tbcvl) [inline demo/accounts.py]: :+1:

File: pydocusign/client.py (hunk 2 above)
Comment (zebuline) [inline pydocusign/client.py]: cf https://github.com/docusign/recipe-012-multiple-docs-python/blob/master/app/py_012_multiple_docs_lib.py#L69:L81 :)

File: demo/accounts.py [hunk 4]
```diff
@@ -80,13 +80,13 @@ def prompt(environ_key, description, default):
 
 # Create a subsidiary account.
 print("3. POST /accounts")
-account_email = "benoit.bryon+pydocusign-test@novapost.fr"
+account_email = "benoit.bryon+pydocusign-test@people-doc.com"
```
Comment (bersace) [inline demo/accounts.py]: En fait, c'est un peu compliqué cette partie. C'est un accès perso au banc de test docusign.

File: demo/accounts.py [hunk 5]
```diff
@@ -9,6 +9,11 @@
 
 import pydocusign
 
+try:
+    raw_input
+except:
```
Comment (bersace) [inline demo/accounts.py]: except NameError ?

File: demo/accounts.py [hunk 6]
```diff
@@ -55,6 +60,18 @@ def prompt(environ_key, description, default):
     'DOCUSIGN_TEST_SIGNER_RETURN_URL',
     'Signer return URL',
     '')
+account_email = prompt(
```
Comment (bersace) [inline demo/accounts.py]: :+1:

**Comment by zebuline** [comment]
A last review ? OPM ?

**Comment by k4nar** [comment]
LGTM :+1:

**Pull request merged**